packages/runtime/test/test_styles.ml

```diff
@@ -331,28 +331,34 @@ let selector_nested_with_mq_and_declarations =
   let classname =
     [%cx
       {|
-      li {
-        list-style-type: none;
-
-        ::before {
-          position: absolute;
-          left: -20px;
-          content: "✓";
-        }
-
+      .a {
+        color: red;
         @media $(mobile) {
-          position: relative;
+          height: 1px;
+          .d {
+            color: green;
+          }
+        }
+        .b {
+          color: blue;
+          @media $(mobile) {
+            height: 2px;
+            .c {
+              color: yellow;
+            }
+          }
         }
       }
     |}]
   in
   let css = get_string_style_rules () in
   assert_string css
```

Comment:
Can you keep the test and add another case instead?

Reply:
Sure!